src/lib/validation/formValidators.ts

```
@@ -138,37 +138,37 @@ export function createFieldValidator(
           return { isValid: false, error: 'Must be a string' };
         }
         break;
-      
+
       case 'number':
         const numValue = Number(value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)